.idea/runConfigurations/composer_update___dev.xml

```diff
@@ -0,0 +1,26 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="composer update --dev" type="docker-deploy" factoryName="docker-image" server-name="Docker">
+    <deployment type="docker-image">
+      <settings>
+        <option name="imageTag" value="srcoder/development-php:php74-fpm" />
+        <option name="command" value="update" />
+        <option name="containerName" value="symbiont_composer" />
+        <option name="entrypoint" value="composer" />
+        <option name="commandLineOptions" value="-u 1000:1000 -w /opt/symbiont" />
```

Comment:
@CyberSecutor I'm assuming here you will have UID 1000 and GID 1000
If at all possible, I would prefer to do this dynamically, but where this is invoked does not run a normal shell and so I can't open a sub shell and run `id -u $USER` and `id -g $USER`  instead.

Had a look at what environment variables are normally available, but no candidates there.

Ofcourse, for now it's pretty safe to assume this value and if not, a developer may always copy this and make a non-project specific configuration of their own.

Reply:
I think thats a pretty safe assumption.